devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/AbstractQuarkusExtension.java

```diff
@@ -117,7 +117,8 @@ protected Manifest manifest() {
         return baseConfig().manifest();
     }
 
-    protected EffectiveConfig buildEffectiveConfiguration(ResolvedDependency appArtifact) {
+    protected EffectiveConfig buildEffectiveConfiguration(ResolvedDependency appArtifact,
+            Map<String, String> platformProperties) {
```

Comment:
A minor comment, it looks like you could pass an `ApplicationModel` here now? Every invocation seems to be obtaining the two arguments from it.